src/pivot-table/components/cells/utils/__tests__/get-adjuster-color.test.ts

```diff
@@ -15,7 +15,7 @@ describe("get-adjuster-color", () => {
         background: Colors.Transparent,
       },
       grid: {
-        border: Colors.Black15,
+        border: "rgba(0, 0, 0, 0.15)",
```

Comment:
I think `Colors.DividerLight` here? 
because it's already been replaced in line 25 in related test case

Reply:
No I could not change that. The tests expected the border color to be transparent. So had to keep that value here. The scenario/s are still supported, just that the default colors are not the same any more.